pkg/app/app_list_test.go

```diff
@@ -277,10 +278,11 @@ releases:
 
 	expectNoCallsToHelm(app)
 
-	out := testutil.CaptureStdout(func() {
+	out, err := testutil.CaptureStdout(func() {
 		err := app.ListReleases(cfg)
 		assert.Nil(t, err)
 	})
+	assert.Nil(t, err)
```

Comment:
Nit but shall we use NoError consistently? 🐱 

```suggestion
	assert.NoError(t, err)
```